<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1355</width>
    <height>879</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="25">
      <item>
       <widget class="QLabel" name="label_1">
        <property name="text">
         <string>Interfaz gráfica</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,5,1,3,2,12,2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Algoritmo:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBoxAlg">
        <item>
         <property name="text">
          <string>Algebraico</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Fourier</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>FBP</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Filtro:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBoxFilt">
        <item>
         <property name="text">
          <string>Ram-Lak</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Shepp-Logan</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Coseno</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Hann</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Hamming</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="reconstruir">
        <property name="text">
         <string>reconstruir</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="abrir">
        <property name="text">
         <string>abrir</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="10,0,10">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,10,0,1,10">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Imagen original:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="imagen">
          <property name="sizeConstraint">
           <enum>QLayout::SetNoConstraint</enum>
          </property>
         </layout>
        </item>
        <item>
         <widget class="Line" name="line_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Reconstrucción:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="reconstruccion"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout" rowstretch="1,10,1,10,1,10" columnstretch="10,0,10" columnminimumwidth="0,0,0">
        <item row="1" column="2">
         <layout class="QVBoxLayout" name="fft"/>
        </item>
        <item row="5" column="0">
         <layout class="QVBoxLayout" name="nada1"/>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>FFT</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <layout class="QVBoxLayout" name="filtro"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Sinograma:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Filtro:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <layout class="QVBoxLayout" name="proyeccion"/>
        </item>
        <item row="1" column="0">
         <layout class="QVBoxLayout" name="sinograma"/>
        </item>
        <item row="5" column="2">
         <layout class="QVBoxLayout" name="nada2"/>
        </item>
        <item row="2" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="10,0">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Proyección:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBox">
            <property name="maximum">
             <number>180</number>
            </property>
            <property name="value">
             <number>180</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
